### DISCUSSION TITLE
Python: 00-getting-started.ipynb object KernelContext can't be used in 'await' expression

### DISCUSSION THREAD

**Discussion by JackYinpei** [microsoft/semantic-kernel#4837] [Q&A] [answered]
"Hi there,
I'm currently working with the semantic-kernel repository and trying to familiarize myself with its functionalities. As I was going through the `00-getting-started.ipynb` notebook, I encountered an issue in the execution of the last code cell.

The error message I received was: `object KernelContext can't be used in 'await' expression.`

I'm fairly new to this repository and I'm not entirely sure of the best way to troubleshoot this problem. I would like to open a discussion on GitHub to ask for assistance, but I want to make sure I explain my issue clearly.

Could someone guide me on how to properly frame this problem for the GitHub discussion in order to get the most efficient help possible?

By the way I am quite sure that my OpenAI key has quota and is correct.

Here is my code 
```
import semantic_kernel as sk
kernel = sk.Kernel()
from semantic_kernel.connectors.ai.open_ai import OpenAIChatCompletion

api_key, org_id = sk.openai_settings_from_dot_env()

kernel.add_chat_service(
    "chat-gpt",
    OpenAIChatCompletion(ai_model_id="gpt-3.5", api_key=api_key, org_id=org_id),
)
plugin = kernel.import_semantic_plugin_from_directory("../../samples/plugins", "FunPlugin")
joke_function = plugin["Joke"]

print(await joke_function("time travel to dinosaur age"))
```

OS: Windows 10
Python: Python 3.10.11
SK: python -m pip install semantic-kernel==0.4.7.dev0

Thank you in advance for your time and assistance.
Best regards

![image](https://github.com/microsoft/semantic-kernel/assets/49555788/2d26488e-b62f-4ffc-9916-9ab779df986b)

**Comment by moonbox3** [comment, marked as answer]
Hi @JackYinpei, please grab the latest package/code. This has since been fixed. (For the future: do create a GitHub issue for this type of thing, please. It'll have more eyes on it versus the Q&A discussions. Thank you.)
